src/System.Data.SqlClient/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.4.0.0</AssemblyVersion>
```

Comment:
You also need to set the PackageVersion here, similar to https://github.com/dotnet/corefx/blob/release/2.0.0/src/System.Data.SqlClient/dir.props#L5. Set it 4.6.0 and update the version in master to 4.7.0.

Reply:
@weshaggard Thanks. Updated it. Will update the master after this pr is merged in another pr.

Reply:
@AfsanehR please go ahead and make a PR for master to update these versions.